### PR TITLE
README edits

### PR DESCRIPTION
**Pull request by LappleApple**
Just a few things: 

- Under "Install and Config section: "By default, pg_view assumes that it can connect to a local PostgreSQL instance with the user postgres and no password. Some systems might require you to change your pg_hba.conf file or set the password in .pgpass. You can use (-c) to specify a different user name in the configuration file." Should it be .... user "postgres" ? and should (-c) actually be in code format?

- Oleksii was interested in providing a screenshot update 

- Under "Usage," see python pg_view --help. A bit of context would be useful here, I think. 

- Under "postgres processes," "type": It ends with the words," all in the console one." That's a bit unclear.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,64 +11,75 @@ pg_view
...
+==============
+
+To run pg_view, you’ll need:
+
+- Linux 2.6
+- Python >= 2.6
+- psycopg2
+- curses
+
+By default, pg_view assumes that it can connect to a local PostgreSQL instance with the user postgres and no password. Some systems might require you to change your pg_hba.conf file or set the password in .pgpass. You can use (-c) to specify a different user name in the configuration file, although specifying the file will turn off autodetection of connection parameters and available databases.
```
Comment (alexeyklyukin) [inline README.rst]: I think the last sentence should be rephrased and extended, i.e:
"You can override the default user name with ``-U`` command-line option or by setting the ``user`` key in the configuration file (see below)."

File: README.rst [hunk 2]
```diff
@@ -11,64 +11,75 @@ pg_view
...
 
-The program queries system /process information files once every tick (by default the tick is 1s). It also
-runs some external programs, like df or du to get filesystem information. The latter might put an extra
-load on a disk subsystem.
+==============
+How pg_view Works:
```
Comment (alexeyklyukin) [inline README.rst]: works should be un-capitalized.

File: README.rst [hunk 3]
```diff
@@ -11,64 +11,75 @@ pg_view
...
+==============
+
+pg_view queries system/process information files once per second. It also runs external programs — df, du, etc. — to obtain filesystem information. Please note that the latter function might add an extra load to your disk subsystem.
```
Comment (alexeyklyukin) [inline README.rst]: The second sentence describes the outdated version of pg_view. We should change it to:
"It also queries the filesystem to obtain the postgres data directory and xlog usage statistics."

File: README.rst [hunk 4]
```diff
@@ -11,64 +11,75 @@ pg_view
...
-* read /proc/ filesystem and detect pid files for the postmaster processes
-* get the working directories from the symlink at /proc/pid/cwd
-* get to the working directories and read PG_VERSION for PostgreSQL verions. If we can't, assume it's not a PostgreSQL directory and skip.
-* try to get all sockets the process is listening to from /proc/net/unix, /proc/net/tcp and /proc/net/tcp6
-* if that fails and version is 9.1 or above, read connection arguments from postmaster.pid
-* check all arguments, picking the first one that allows us to establish a connection
-* if we can't get either the port/host or port/socket_directory pair, bail out.
+- reads /proc/ filesystem and detects pid files for the postmaster processes
+- gets the working directories from the symlink at /proc/pid/cwd
+- reads the PG_VERSION for PostgreSQL versions (If it doesn’t, assume it's not a PostgreSQL directory, and skip)
```
Comment (alexeyklyukin) [inline README.rst]: if it doesn't **exist** (also, why is "If" capitalized?)

File: README.rst [hunk 5]
```diff
@@ -84,103 +95,72 @@ is missing from the database-specific section. For instance::
...
-            amount of data read or written from the partition (in MB/s).
-    * age
-            time from the process start
-    * db
-            the database the process runs on
-    * query
-            the query the process executes.
+==============
 
+see ``python pg_view --help``
```
Comment (alexeyklyukin) [inline README.rst]: You can get a short description of available configuration options with ``pg_view --help``.

File: README.rst [hunk 6]
```diff
@@ -84,103 +95,72 @@ is missing from the database-specific section. For instance::
...
+    - **type**: either containing database data (data) or WAL (xlog).
+    - **until_full**: the time remaining before the current partition will run out of space, *if* we only consider writes to the corresponding data directory (``/data`` or ``/pg_xlog``). This column is only shown during the warning (3h) or critical (1h) conditions, and only considers momentary writes. If a single process writes 100MB/s on a partition with 100GB left for only two seconds, it will show a critical status during those two seconds.
+- **postgres processes**
+    - **age**: length of time since the process started.
+    - **db**: the database the process runs on.
+    - **query**: the query the process executes.
+    - **read, write**: The amount of data read or written from the partition in MB/s.
+    - **s**: process state. ``R`` - 'running', ``S`` - 'sleeping', ``D`` - 'uninterruptable sleep'; see ``man ps`` for more details.
+    - **type**: either a system process (autovacuum launcher, logger, archiver, etc.) or a process that executes queries (backend or autovacuum). By default, only user processes are shown in curses mode (press 's' to show all of them), and all in the console one.
```
Comment (alexeyklyukin) [inline README.rst]: By default, only user processes are shown in curses output mode (press 's' to add the system processes). Both system and user processes are shown in the console mode.

**Comment by muzemmil-dotcom** [comment]
002

**Pull request merged**